pkg/database/sqlite_db_instance.go

```diff
@@ -0,0 +1,162 @@
+package database
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+	glog "log"
+	"net/url"
+	"os"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/rotisserie/eris"
+	"golang.org/x/exp/slices"
+
+	lru "github.com/hashicorp/golang-lru/v2"
+	"github.com/mattn/go-sqlite3"
+	log "github.com/sirupsen/logrus"
+	"gorm.io/driver/sqlite"
+	"gorm.io/gorm"
+	"gorm.io/gorm/logger"
+	"gorm.io/plugin/dbresolver"
+)
+
+const (
+	SQLiteCustomDriverName = "sqlite3_custom_driver"
+)
+
+// SqliteDbInstance is the sqlite specific variant of DbInstance
+type SqliteDbInstance struct {
+	DbInstance
+}
+
+// Reset behavior for this instance
+func (f SqliteDbInstance) Reset() error {
+	return eris.New("unable to reset sqlite database")
```

Comment:
emm, why we return an error? Won't it be a problem for us? Maybe return:
```
return eris.New("reset operation not supported yet by sqlite")
```
what do you think?

Reply:
that was the previous behavior from database.go, but will make message change.

Reply:
In fact, the original `resetDB` is only called from the postgres initialization path, so it cannot be invoked -- it's a guardrail, not expected to be invoked.